extension/search/caniuse.js

```diff
@@ -0,0 +1,34 @@
+function CaniuseSearch(index) {
+  this.feats = {};
+  index.forEach(([ver, flag, title, rfc]) => {
+    let searchTerm = `${flag}: ${title}`.replace(/[-_\s#\?]/ig, "");
+    this.feats[searchTerm] = { ver, flag, title, rfc };
+  });
+  this.searchTerms = Object.keys(this.feats);
+}
+
+CaniuseSearch.prototype.search = function (rawKeyword) {
+  let keyword = rawKeyword.replace(/[-_\s#\?]/ig, "");
+  let result = [];
+
+  for (let searchTerm of this.searchTerms) {
+    if (searchTerm.length < keyword.length) continue;
+
+    let foundAt = searchTerm.indexOf(keyword);
+    if (foundAt > -1) {
+      let rfc = this.feats[searchTerm].rfc;
+      // skip those without RFC when searching for RFCs
+      if (!(rawKeyword.startsWith("??") && rfc == null)) {
```

Comment:
I think `rawKeyword.startsWith("??")` is redundant as we replace `?` at line 11.

Reply:
so the idea here is we only skip these if we're searching for RFCs, hence the check for `??`, and also that's why I used `keyword` vs. `rawKeyword`.

Reply:
Ok, got it 👍